nemo/collections/asr/models/clustering_diarizer.py

```diff
@@ -365,6 +383,78 @@ def path2audio_files_to_manifest(self, paths2audio_files, manifest_filepath):
                 entry = {'audio_filepath': audio_file, 'offset': 0.0, 'duration': None, 'text': '-', 'label': 'infer'}
                 fp.write(json.dumps(entry) + '\n')
 
+    @experimental
+    def parse_scale_configs(self, window_lengths_in_sec, shift_lengths_in_sec, multiscale_weights):
```

Comment:
Can we move this function to speaker utils please?

Reply:
How about move it to diarization utils?

Reply:
Okay now it's unclear to me what should be belongs to diarization_utils and what should be belongs to speaker_utils